### PR TITLE
Release/v1.2 - Launcher instance (ARTP-900)

### PR DESCRIPTION
**Pull request by mattydono**
Fix: uuid is now unique to the environment launched

This should allow us to launch a single openfin instance per env.

### CODE REVIEW

File: src/client/src/apps/utils/openfin-utils.ts [hunk 1]
```diff
@@ -31,25 +32,27 @@ async function restoreExistingApp(existingApp: Application): Promise<void> {
...
   name,
   url,
+  uuid,
   windowOptions,
 }: ApplicationConfig): Promise<Application> {
   const appOptions: fin.ApplicationOption = {
     name,
     url,
-    uuid: name,
+    uuid: uuid!,
```
Comment (mozmorris) [inline src/client/src/apps/utils/openfin-utils.ts]: Thinking about this now @mattydono, you probably want to fallback to the original `name` arg. e.g `uuid: uuid || name,`

File: src/client/package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
   },
   "scripts": {
     "precommit": "lint-staged",
-    "start": "cross-env REACT_APP_BUILD_VERSION=$(git describe) react-scripts start",
```
Comment (mozmorris) [inline src/client/package.json]: @rikoe I take it we're happy for these other changes to find their way into this release branch?

File: src/client/src/apps/utils/openfin-utils.ts (hunk 1 above)
Comment (mattydono) [reply src/client/src/apps/utils/openfin-utils.ts]: ah yes that makes a lot more sense!

File: src/client/package.json (hunk 2 above)
Comment (rikoe) [reply src/client/package.json]: yeah that's fine

**Pull request merged**